model/actors/init/idaddress.go

```diff
@@ -38,6 +44,14 @@ func (ias IdAddressList) PersistWithTx(ctx context.Context, tx *pg.Tx) error {
 }
 
 func (ias IdAddressList) Persist(ctx context.Context, db *pg.DB) error {
+	ctx, _ = tag.New(ctx, tag.Upsert(metrics.TaskNS, tasks.InitActorPoolName))
+	stats.Record(ctx, metrics.TaskQueueLen.M(-1))
+
+	start := time.Now()
+	defer func() {
+		stats.Record(ctx, metrics.PersistDuration.M(metrics.SinceInMilliseconds(start)))
+	}()
```

Comment:
It would be great it this was a timer thingy in the `metrics/` package so we could do things like:
```go
mt := NewMetricTimer()
defer mt.Stop(ctx)
```